corehq/apps/analytics/signals.py

```diff
@@ -132,4 +132,4 @@ def track_user_login(sender, request, user, **kwargs):
             return
 
         meta = get_meta(request)
-        track_user_sign_in_on_hubspot.delay(couch_user, request.COOKIES, meta)
+        track_user_sign_in_on_hubspot(couch_user, request.COOKIES, meta, request.path)
```

Comment:
:+1: if you add `CELERY_ALWAYS_EAGER= True` to your localsettings I don't think you need to do that.